pkg/render/dex_config.go

```diff
@@ -479,6 +479,25 @@ func (d *dexConfig) Connector() map[string]interface{} {
 			"insecureSkipEmailVerified": d.authentication.Spec.OIDC.EmailVerification != nil &&
 				*d.authentication.Spec.OIDC.EmailVerification == oprv1.EmailVerificationTypeSkip,
 		}
+		promptTypes := d.authentication.Spec.OIDC.PromptTypes
+		if promptTypes != nil {
+			length := len(promptTypes)
+			prompts := make([]string, length)
+			for i, v := range promptTypes {
+				switch v {
+				case oprv1.PromptTypeNone:
+					prompts[i] = "none"
+				case oprv1.PromptTypeSelectAccount:
+					prompts[i] = "select_account"
+				case oprv1.PromptTypeLogin:
+					prompts[i] = "login"
+				case oprv1.PromptTypeConsent:
+					prompts[i] = "consent"
+				}
+			}
+			// RFC specifies space delimited case sensitive list: https://openid.net/specs/openid-connect-core-1_0.html#AuthRequest
+			config["promptType"] = strings.Join(prompts, " ")
+		}
```

Comment:
Minor: The Connector function is starting to grow in size. Might be easier to read to split case into a separate function (at some point).

Reply:
I'd like to defer this for now.